content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -235,19 +236,19 @@ Finally to add some nice enhancements, we'll use:
 - [getmdl.io](http://getmdl.io) to syle the components;
 - [pusher.com](http://pusher.com) to automatically update our app when new messages arrive.
 
-With these modifications and without having to write any backend code, we have a WebApp accessible at this URL: [https://enguerran.github.io/kintochat/#4815162342](https://enguerran.github.io/kintochat/#4815162342).
+With these modifications and without having to write any backend code, we have our chat WebApp accessible at this URL: [https://enguerran.github.io/kintochat/#4815162342](https://enguerran.github.io/kintochat/#4815162342).
 
 The example [source code is available on Github](https://github.com/enguerran/kintochat), please keep in mind that it is just for educational purpose, it’s not production ready, it’s not even covered with tests.
 
 *N.B.: if you prefer to use Elm instead of JavaScript, you can [find it here](https://github.com/Natim/kinto-elm-chat).*
 
 ## But, what is Kinto?
 
-You can think of Kinto as a backend API service to store and sync your data.
+You can think of Kinto as a generic backend to store and sync your data.
 
-Kinto is written in Python around [Pyramid](https://trypyramid.com/). Kinto is highly configurable through its `kinto.ini` file and [a set of plugins](https://github.com/Kinto/kinto/wiki/Plugins).
+Kinto is written in Python around [Pyramid](https://trypyramid.com/). Kinto is highly configurable through its `kinto.ini` file and [plugins](https://github.com/Kinto/kinto/wiki/Plugins).
 
-Kinto is [OpenSource](https://github.com/kinto) and allows us to store of our user's data away from Google, Amazon, Apple or Microsoft closed silos and other private cloud unicorns.
+Kinto is [OpenSource](https://github.com/kinto) and allows us to store of our user's data away from GAFAM closed silos and other private cloud unicorns.
```

Comment:
Ideally a link to some definition for GAFAM would be welcome. The thing is it seems it's mostly used by French speaking people... Still unsure we should use it here.